manifests/04-rbac-rolebinding.yaml

```diff
@@ -94,3 +94,31 @@ subjects:
   - kind: ServiceAccount
     name: console
     namespace: openshift-console
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator-monitoring
```

Comment:
I think we can just name it `console-operator`. its the binding for the `console-operator` and lists the roles it binds (which should all be named something specific, I believe).  The one we have in `openshift-config` is just `console-operator`.